images/dashboard/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# TODO(jlewi): How can we make it work with golang:1.8.2-alpine
```

Comment:
Do we want a separate Docker image for the dashboard?

Reply:
I don't have a strong opinion on this. having a single image would make the release scripts easier to maintain. I don't see any obvious drawbacks.